toolbox

```diff
@@ -490,6 +490,7 @@ images_get_details()
                 echo "$base_toolbox_command: failed to get details for image $image" >&2
                 return 1
             fi
+            echo
```

Comment:
Is this really needed? Or is it a spurious change? It seems to work without it, but I am not sure if I am missing something.

Reply:
Yes, this is needed at least on podman 1.0.1, but looks like we can avoid it with 1.2.0 at least. Do we have a minimal podman version we support here?

Reply:
Yes, I confirm it works without this and with podman 1.2.0.

Reply:
Ok, thanks for checking!

It's a good question - what's the minimum supported Podman version. :) I don't know. We either avoided using anything that needed a newer Podman (eg., `podman exec --workdir` from https://github.com/containers/libpod/pull/2097), or we added a runtime check for it (eg., `podman create --dns=none --no-hosts` from https://github.com/containers/libpod/pull/2747).

There is no real reason for it, other than the fact that rootless Podman has been growing and maturing along with the Toolbox. So not having a hard requirement made it easier to revert to an older Podman to work around regressions, `git bisect`, implement something before Podman had support for it, and so on.

But it's possible that we are already hard-depending on features added to a certain version of Podman, and that we can exploit that to drop some of our compatibility code. If you find something like that then please submit a pull request.